src/Mapbender/ManagerBundle/Resources/public/js/application-edit.js

```diff
@@ -324,6 +319,31 @@ $(function() {
         return false;
     });
 
+    function elementFormSubmit() {
+        var form = $("#elementForm"),
```

Comment:
This is just a personal favor, but maybe prefix elements in their jQuery variant with a `$`.